usaspending_api/etl/award_helpers.py

```diff
@@ -119,3 +119,35 @@ def update_contract_awards(award_tuple=None):
         rows = cursor.rowcount
 
     return rows
+
+
+def update_award_subawards(award_tuple=None):
```

Comment:
I wanted to keep all the SQL helpers together so that hopefully we can refactor them all together in the future.